src/components/PageWrapper/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
This component is used to setup the "sticky footer". It is a flex box container that wraps each page so it can push the footer to the bottom of the viewport or at the bottom of the scroll